requirements_examples.txt

```diff
@@ -0,0 +1 @@
+matplotlib
```

Comment:
Only dependency I found for examples, though I suspect others may be added later?